tests/test_utils.py

```diff
@@ -216,6 +218,48 @@ def test_custom_email_claims(self):
                 security.get_token_email({})
             self.assertEqual(ex.exception.status, 401)
             self.assertEqual(ex.exception.message, 'Authorization token is missing email claims.')
+    def test_multipart_parallel_upload(self):
+        size = 7 * 1024 * 1024
+        data = os.urandom(size)
+        metadata = {'something': "foolish"}
+        part_size = 5 * 1024 * 1024
+        s3_client = Config.get_native_handle(Replica.aws)
+        bucket = os.environ['DSS_S3_BUCKET_TEST']
+        with self.subTest("copy multiple parts"):
```

Comment:
Are the uploaded files cleaned up?

Reply:
Cleaned up by kharon in dev, other stages soon.